.gitignore

```diff
@@ -20,4 +20,6 @@ tests/benchmarks/.benchmarks/
 *.output
 *.pdf
 *.pyc
+*.swp
```

Comment:
what are these? the vim swp files?

Reply:
Yes. I am assuming it won't break anything.

Reply:
yea it's fine just was wondering